### PR TITLE
fix for https://github.com/chef/chef/issues/2113

### PR DESCRIPTION
**Pull request by dansteen**

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by dansteen** [comment]
Looks like the test above failed due to git connectivity issues:

    384.75s$ git clone --depth=50 git://github.com/chef/chef.git chef/chef
    Cloning into 'chef/chef'...
    fatal: unable to connect to github.com:
    github.com[0: 192.30.252.131]: errno=Connection timed out
    The command "eval git clone --depth=50 git://github.com/chef/chef.git chef/chef" failed. Retrying, 2 of 3.

**Comment by smith** [comment]
@dansteen I restarted the build, so it should go through now. Thanks!

**Comment by lamont-granquist** [comment]
this clause has a typo and is a bit wordy:

```
'Did you perhapse intend to use --secret-file rather than --secret?'
```

maybe

```
`You may need to use --secret-file rather than --secret`
```

**Comment by smurawski** [comment]
Fix the type and I'm :+1:

**Comment by thommay** [comment]
thanks, @dansteen
